src/foam/nanos/auth/services.jrl

```diff
@@ -194,9 +207,11 @@ p({
   "name":"languageDAO",
   "lazy":false,
   "serve":true,
+  "authenticate": false,
```

Comment:
remove authenticate: false

Reply:
@jlhughes does this need to be removed?  countryDAO is also authenticate: false

Reply:
@adamvy languageDAO and countryDAO nspecs should be 
authenticate: false ( when true, the user must have service._nspec-id_ permission)
along with 
EasyDOA..
setAuthorize(true)  

EasyDAO with authorize: true will use the GlobalReadAuthorizer (all can read, permission required to update), which is what we want. 
The setAuthorizer is optional if the prefix  that was used for permissions was not the 'of' in lowercase. 
So for both language and country models, neither require the explicit setAuthorizer(...). 

But they are both correct as they stand.